core/translations/fr.json

```diff
@@ -315,6 +315,14 @@
     "ethereum__units_template": "{} unités",
     "ethereum__unknown_token": "Jeton inconnu",
     "ethereum__valid_signature": "Signature valide.",
+    "ethereum__staking_stake": "STAKE",
+    "ethereum__staking_stake_address": "ADRESSE DE STAKING",
+    "ethereum__staking_stake_intro": "Staker de l'ETH sur Everstake?",
+    "ethereum__staking_unstake": "UNSTAKE",
+    "ethereum__staking_unstake_intro": "Terminer le Staking de l'ETH sur Everstake?",
+    "ethereum__staking_claim": "RETIRER",
+    "ethereum__staking_claim_address": "ADRESSE DE RETRAIT",
+    "ethereum__staking_claim_intro": "Retirer l'ETH d'Everstake?",
```

Comment:
should we autotranslate these new strings? or should we leave them for the translators?
cc @Hannsek